.github/workflows/sync-smartview.yaml

```diff
@@ -0,0 +1,83 @@
+name: Sync smartview API spec
+
+on:
+  workflow_dispatch:
+
+jobs:
+  sync:
+    strategy:
+      matrix:
+        go-version: [1.19.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Get current date
+      id: date
+      run: echo "date=$(date +'%Y-%m-%d')" >> "$GITHUB_OUTPUT"
+    - name: Install Go
+      uses: actions/setup-go@v5
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: GitHub user
+      run: |
+        # https://api.github.com/users/github-actions[bot]
+        git config user.name 'github-actions[bot]'
+        git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
+    - name: Fetch latest spec
+      id: fetch
+      run: |
+        make -f Makefile.smartview fetch
+        git add spec/services/smartview
+        echo `git commit -m 'sync: fetch ${{ steps.date.outputs.date }} spec and apply patches'`
+    - name: Apply spec patches
+      id: patch
+      if: ${{ always() && steps.fetch.conclusion == 'success' }}
+      run: |
+        make -f Makefile.smartview patch
```

Comment:
We'll want to `combine` before patching. We would want to be careful about the order of the source URLs being combined so that the fetched+combined spec can be patched against.